### ISSUE TITLE
Train Generator Network

### ISSUE THREAD

**Issue by CristobalBL** [chanshing/geocondition#1]
Hello,

Thanks for this work, very interesting.

I would like to know if you have an example of how to train the Generator Network used in this work that you are code with PyTorch. To try other types of images.

**Comment by chanshing** [comment]
Thanks for your interest. For the generator network, we simply used the WGAN model https://github.com/martinarjovsky/WassersteinGAN with bare minimum tweaks, although you could use more recent methods as GAN research moves very quickly. You can find more on the training and evaluation of the generator network in our parallel work https://www.frontiersin.org/articles/10.3389/frwa.2020.00005/full but in summary, we just used WGAN with default hyperparameters except learning rate and batch size.

**Comment by CristobalBL** [comment]
Thanks for the reply, last week i can check with my implementation. From the link you gave me, WGAN worked perfect.
